content/articles/coming-soon-landing-page-using-javascript-timers/index.md

```diff
@@ -0,0 +1,403 @@
+
+**syntanx**
+
+`setTimeout(function, milliseconds)`.
+
+You must give two parameters to utilize this function:A Function parameter that describes the function to be executed, and an optional Delay parameter that determines how long to wait before the function is executed (1 second = 1000 milliseconds).
+
+**Example**
+
+```javascript
+<script>
+function delayGreetings() {
+    alert('Hello World!');
+}
+</script>
+<button type="button" onclick="setTimeout(delayGreetings,2000)">click Me</button>
+```
+
+**when the button is clicked the greetings will be displayed after 2 seconds**
+
+**NOTE :** If the delay parameter is 0 or if it is not specified, the function is executed immediately.
+
+_TASK-1_
+
+Use `setTimeout` function to print the following two messages after their respective delays .
+
+- After 4 seconds, display greetings "Hello FabulousDesigns after 4secs"
+- After 8 seconds, display greetings "Hello FabulousDesigns after 8secs."
+
+**CONSTRAIT :**
+In your solution, you can only define one function, which includes inline functions. As a result, a lot of setTimeout calls will have to use the same code.
+
+**SOLUTION**
+
+- **Task-1** may be solved in a variety of ways, but this is one of them.
+
+```javascript
+  const delayGreetings = (delay) => {
+    console.log("Hello FabulousDesigns after " + delay + " seconds");
+  };
+  setTimeout(delayGreetings, 4 * 1000, 4);
+  setTimeout(delayGreetings, 8 * 1000, 8);
+  ```
+
+#### Explanation
+
+- I gave delayGreetings a delay parameter and utilized the delay argument's value in the displayed message. As a result, depending on the delay value we provide to the function, it might print different messages.
+
+- After that, I used delayGreetings function in two setTimeout calls, one for 4 seconds and the other for 8 seconds. A third parameter is sent to both of these setTimeout calls to represent the delay argument for delayGreetings.
+
+### Code Execution At Regular Intervals
+
+---
+
+Similarly, the setInterval() method may be used to execute a function or a given piece of code at certain intervals.
+
+**syntanx**
+
+`setInterval(function, milliseconds)`.
+
+Begin by defining two parameters, one of which defines the function to be performed and another which provides the period (one second = 1000 milliseconds) in which to wait until the function is executed.
+**Example**
+
+```javascript
+setInterval(
+  () => console.log("Hello fabulousDesigns after every 3 seconds"),
+  3000
+);
+```
+
+The above code executes after every 3 seconds.
+
+_Task-2_
+
+- use setInterval function to display your local computer time after every one second.
+
+  **Solution**
+
+- one of the way to sovle **Task-2** is:
+
+```javascript
+<script>
+function displayTime() {
+    var d = new Date();
+    document.querySelector("#clock").innerHTML = d.toLocaleTimeString();
+}
+setInterval(displayTime, 1000);
+</script>
+
+<p>current time on your computer is: <span id="clock"></span></p>
+```

```

Comment:
This code cannot work. Ensure you tell user which file to create. Ensure your outputs are captures as images and embedded on this article.